docs/extend/extend-apiml/api-mediation-infinispan.md

```diff
@@ -20,7 +20,7 @@ For more information on Infinispan, see the [official Infinispan documentation](
 Infinispan can be used with both on standalone instance and High Availability mode. In case of multiple Caching Service instances, 
 you will have to specify all the cluster nodes (members). Each Infinispan node is bound to specific Caching Service instance and runs on a different port and host, which can be configured. See the [infinispan configuration](#infinispan-configuration) to know how to configure multiple Infinispan nodes.
```

Comment:
"... you must specify all the cluster nodes (members)."

Reply:
I made a mistake when I created this branch and, as a result, it now includes another writer's content. Since this is the work of another writer, I will not be editing this file.